src/MICore/JsonLaunchOptions.cs

```diff
@@ -95,6 +95,12 @@ public abstract partial class BaseOptions
         /// </summary>
         [JsonProperty("setupCommands", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public List<SetupCommand> SetupCommands { get; protected set; }
+
+        /// <summary>
+        /// Optional parameter. If true, the debugger should stop after connecting to the target.
+        /// </summary>
+        [JsonProperty("stopAtConnect", DefaultValueHandling = DefaultValueHandling.Ignore)]
```

Comment:
You have this in `BaseOptions` so that it is common between attach and launch, but does it actually apply to attach?

Reply:
Moved to `LaunchOptions`.